packages/studio-ui/src/web/components/Layout/TopNav/cloud/pat-input.tsx

```diff
@@ -0,0 +1,52 @@
+import { Alignment, AnchorButton, Button, InputGroup, Tooltip } from '@blueprintjs/core'
+import { lang } from 'botpress/shared'
+import React from 'react'
+
+import style from './style.scss'
+
+export function PatInput(props: {
+  pat: string
+  valid: boolean
+  loading?: boolean
+  onChange: (pat: string) => void
+  onSave: () => void
+}): JSX.Element {
+  const { pat, valid, loading, onChange, onSave } = props
+
+  return (
+    <div>
+      <div className={style.patContainer}>
+        <InputGroup
+          className={style.patInput}
+          placeholder={lang.tr('topNav.deploy.enterPersonalAccessToken')}
+          value={pat}
+          onChange={(e) => onChange(e.target.value)}
+          rightElement={
+            valid ? (
+              <Button disabled={true} icon={'tick-circle'} minimal={true} />
```

Comment:
I don't know if you wanted to be explicit or because you didn't know but you don't have to write `={true}` when you want a boolean prop to be true. in this case 

```html
<Button disabled minimal icon="tick-circle"/>
```

Reply:
Fixed in 4355c311a19e6fafa42eaa945bf995b3c4b7c65e